pyomo/repn/tests/ampl/test_ampl_comparison.py

```diff
@@ -98,16 +98,20 @@ def nlwriter_asl_test(self, name):
         pass
 
     # obtain the nl file summary information for comparison with ampl
-    p = pyutilib.subprocess.run(
-        'gjh_asl_json '+currdir+name+'.test.nl rows='
-        +currdir+name+'.test.row cols='+currdir+name+'.test.col')
-    self.assertTrue(p[0] == 0, msg=p[1])
+    p = subprocess.run(['gjh_asl_json',
+                        currdir+name+'.test.nl',
+                        'rows='+currdir+name+'.test.row',
+                        'cols='+currdir+name+'.test.col'],
+                       stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Should we use `stderr=subprocess.STDOUT` so that the test failure message will contain the error?